rest/reference.go

```diff
@@ -128,6 +130,14 @@ func (c *ReferenceClient) ListDividends(ctx context.Context, params *models.List
 	})
 }
 
+func (c *ReferenceClient) ListFinancials(ctx context.Context, params *models.ListFinancialsParams, options ...models.RequestOption) *iter.Iter[models.Financial] {
```

Comment:
we should separate this into its own `VXClient`. i want it to be super clear that users are using something that could have breaking changes. this method also needs to be documented and we should mention that breaking changes (or deprecation) could happen on it

Reply:
i also don't want the top-level client to inherit the `VXClient` so instead of users calling `c.ListFinancials`, i want them to call `c.VX.ListFinancials`

Reply:
Would you like a separate financials.go file, or should I just make a second client in the reference.go file?

Reply:
it should probably be a vx.go file or something along those lines